common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -306,7 +306,10 @@ def _get_selected_child_blocks(self):
         current user.
         """
         for block_type, block_id in self.selected_children():
-            yield self.runtime.get_block(self.location.course_key.make_usage_key(block_type, block_id))
+            child = self.runtime.get_block(self.location.course_key.make_usage_key(block_type, block_id))
+            if child is None:
+                logger.info("Child not found for {} {}".format(str(block_type), str(block_id)))
```

Comment:
What about `logger.info("Child not found for %s %s", str(block_type), str(block_id))`?